src/constants/env.js

```diff
@@ -0,0 +1,9 @@
+const ENV_NAMES = {
```

Comment:
This is not an opinionated question, but I wanted to ask since this is our first constant in the project.

What is the benefit of defining const ENV_NAMES explicitly as opposed to just having something like

```
export default {
  ENV_NAMES: { ... },
}
```

Reply:
No difference to me. Could do it this way. Leaving this open for @reefdog's opinion.

Reply:
For reef's consideration: we can also do the `export const ENV_NAMES` approach here too.

Reply:
True, and that's my preference, but we'd need something else to export, even if an empty default.

Reply:
Ah, for linting right?

Heads up that we regularly use [this pattern]() to handle the linting complaint:

```
// Disabling because we intend to have more exports in the future.
/* eslint-disable import/prefer-default-export */
```

It's a valid linting rule for things like class files, but not so much for utility / constant files that just happen to currently only have one export.

I've been thinking of making it a habit of disabling that rule at the eslintconfig level since 9 out of 10 times I see it complain it's in one of those files.

Reply:
Okay! Thinking about this. Do we intend to have more constants in this file in the future? It's in a file called `env.js`, because it seemed like that fit under a directory called constants. Could either put this in `index.js` or make it the default for this file.

Reply:
Ohhhhhhh that's a good point; and yeah that was the right place for it.

And you've convinced me that export default is the way to go here.  Back to deferring to @reefdog on the `const` vs direct export default {} form

Reply:
I'm happy you both defer to me even though I have absolutely no reason to be the authority on this. But now I've done some Googling and while the [ESLint docs themselves were useless](https://github.com/benmosher/eslint-plugin-import/blob/v2.18.2/docs/rules/prefer-default-export.md) as to reasons, there is [this long debate](https://github.com/airbnb/javascript/issues/1365) that was very useful, as well as [this article from the creator of ESLint](https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/). They've convinced me that we should actually purge default exports in favor of named exports; that article, particularly, was convincing.

I think what I'd like to see is:

1. Disable `prefer-default-export`
2. In fact, can we ban them?
3. Convert all our existing default exports to named. The fact that we currently can accidentally `import IApliance from './IAppliance'` is a _bug_ waiting to bite us; it will _work_ in that file (so long as we keep referencing `IApliance`) but things like project-wide searching will break. Instead, by forcing ourselves to use named exports/imports (`import { IAppliance } from './IAppliance`), we avoid this accident. And if for some reason we ever _really_ want to refer to the exported item differently during a specific import, we can always use `import { IAppliance as IApliance } from './IAppliance'`, which is a very explicit way to signal what we're doing.
4. Tangentially, continue pushing off more code organization to file/directory structure, though not so far as the "one export per file".

So for this PR specifically, I'd say remove the `export default {}` bit at the end, add the rule-disabling line that @slifty noted we regularly use, and then let's open a new issue to purge (or at least discuss purging) default exports.

Reply:
Okay. Will do.